README.rst

```diff
@@ -589,6 +589,87 @@ Django setting ``DJANGO_REDIS_CONNECTION_FACTORY``.
             # creating a new connection using the `get_connection` method.
             pass
 
+Use the sentinel connection factory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to facilitate using `Redis Sentinels`_, django-redis comes with a
+built in sentinel connection factory, which creates sentinel connection pools.
+In order to enable this functionality you should add the following:
+
+
+.. code-block:: python
+    :caption: myproj/settings.py
+
+    # Enable the alternate connection factory.
+    DJANGO_REDIS_CONNECTION_FACTORY = 'django_redis.pool.SentinelConnectionFactory'
+
+    # These sentinels are shared between all the examples, and are passed
+    # directly to redis Sentinel. These can also be defined inline.
+    SENTINELS = [
+        ('sentinel-1', 26379),
+        ('sentinel-2', 26379),
+        ('sentinel-3', 26379),
+    ]
+
+    CACHES = {
+        "default": {
+            "BACKEND": "django_redis.cache.RedisCache",
+            "LOCATION": "redis://service_name/db",
```

Comment:
Just wondering: why do we need a `LOCATION` here? 
I mean the current master should be identified from the sentinels, shouldn't it?

Reply:
This specifies which replica set to ask for when talking to the sentinels. The request to the sentinels will be:

```
# primary
SENTINEL MASTER <service_name>
# replicas
SENTINEL REPLICAS <service_name>
```

I had left the `LOCATION` a normal redis URL because that seemed simpler and allows this code to use a lot of existing code.